### PR TITLE
fewer prismlauncher workarounds

### PR DESCRIPTION
**Pull request by nekopsykose**

### CODE REVIEW

File: _docs/configuration/games.md [hunk 1]
```diff
@@ -47,10 +47,10 @@ the box.
 You can play Minecraft natively on Chimera on the `x86_64` architecture
 (and possibly others with custom JARs).
 
-First, you will need Java. Install OpenJDK 17:
+First, you will need Java. Install OpenJDK 21:
```
Comment (q66) [inline _docs/configuration/games.md:50]: these instructions are actually not correct anymore; the right thing to do is to add `java-jre` (and then you can choose the default via like `java-jre-openjdk17-default` or the likes, but 21 is default right now)

Comment (nekopsykose) [reply _docs/configuration/games.md]: hm, sure

i made it java-jdk cause it was a jdk before. though whatever

Comment (q66) [reply _docs/configuration/games.md]: i think jre should technically be enough, though it does not matter much...

maybe i should add a separate page for java and then we could link to it

Comment (nekopsykose) [reply _docs/configuration/games.md]: sure

**Pull request merged**